internal/pkg/form/form.go

```diff
@@ -112,16 +113,15 @@ func ConfirmDeletion(cmd *cobra.Command, promptMsg, stringToType string) (bool,
 	return false, errors.NewErrorWithSuggestions(fmt.Sprintf(`input does not match "%s"`, stringToType), DeleteResourceConfirmSuggestions)
 }
 
-func ConfirmEnter(cmd *cobra.Command) (error) {
+func ConfirmEnter(cmd *cobra.Command) error {
 	// Note: this function does not check against any regex;
 	// any string is acceptable
-	prompt := NewPrompt(os.Stdin)
-	f := Field{Prompt: "Press enter to continue or Ctrl-C to cancel"}
-	show(cmd, f)
+	utils.Print(cmd, "Press enter to continue or Ctrl-C to cancel:")
 
-	if _, err := read(f, prompt); err != nil {
+	if _, err := term.ReadPassword(int(os.Stdin.Fd())); err != nil {
```

Comment:
You might find it easier to use the `form` package and set `IsHidden = true`. (The current approach is a bit confusing since we're not reading a password)

Reply:
I considered that, but using form with `IsHidden` still prints *'s instead of surpressing output

Reply:
Ah, right. Let's write a comment explaining why we're using this function then!